packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -88,6 +92,30 @@ describe("Garbage Collection Tests", () => {
         clock.restore();
     });
 
+    describe("Session expiry is called", () => {
+        beforeEach(() => {
+            closeCalled = false;
+        });
+        it("Session expiry is called with existing timeout", async () => {
+            const metadata: IContainerRuntimeMetadata = 
+                { summaryFormatVersion: 1, message: undefined, sessionExpiryTimeoutMs: 1 }
+            createGarbageCollector(undefined, undefined, metadata);
+            clock.tick(1);
+            assert(closeCalled, "Close should have been called.");
+        });
+
+        it("Session expiry is called", async () => {
+            const settings = { "Fluid.GarbageCollection.RunSessionExpiry": "true" };
+            const oldRawConfig = sessionStorageConfigProvider.value.getRawConfig;
+            sessionStorageConfigProvider.value.getRawConfig = (name) => settings[name];
+            createGarbageCollector();
+            const expectedTimeoutMs = 30 * 24 * 60 * 60 * 1000; //clock tick 30 days
```

Comment:
You should export this value in garbageCollection.ts and use it. This will make sure the test is in sync and doesn't need updating every time the default value changes.